kratos/tests/sources/test_nodal_data.cpp

```diff
@@ -80,12 +80,10 @@ namespace Kratos {
 					velocity_y_sum += i_node->FastGetSolutionStepValue(VELOCITY_Y);
 				}
 				KRATOS_CHECK_DOUBLE_EQUAL(pressure_sum, i * size);
-				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, i * size);
+				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, 2*i * size);
 				KRATOS_CHECK_DOUBLE_EQUAL(velocity_y_sum, 0.00);
 			}
 		}
```

Comment:
:-( it is actually problematic :-(

see this output
>>> print(DISPLACEMENT)
DISPLACEMENT variable #196
>>> print(DISPLACEMENT_X)
DISPLACEMENT_X component of DISPLACEMENT variable #221
>>> print(DISPLACEMENT_Y)
DISPLACEMENT_Y component of DISPLACEMENT variable #222
>>> print(DISPLACEMENT_Z)
DISPLACEMENT_Z component of DISPLACEMENT variable #223
>>> print(VELOCITY)
VELOCITY variable #881
>>> print(VELOCITY_X)
VELOCITY_X component of VELOCITY variable #903
>>> print(VELOCITY_Y)
VELOCITY_Y component of VELOCITY variable #908
>>> print(VELOCITY_Z)
VELOCITY_Z component of VELOCITY variable #913

DISPLACEMENT_* is consecutive
VELOCITY_* is NOT consecutive